femtosip.py

```diff
@@ -320,16 +320,16 @@ def make_invite_sip_packet(self,
         # Assemble the header fields
         fields = collections.OrderedDict()
         fields['Via'] = (
-            'SIP/2.0/' + self.protocol.upper() + ' ' + self.local_ip + ':' + str(self.port) +
+            'SIP/2.0/' + self.protocol[:3].upper() + ' [' + self.local_ip + ']:' + str(self.port) +
```

Comment:
I'm not sure about the added `[` and `]`. Are you sure that this is supported by all SIP clients, including legacy devices only supporting IPv4?

Reply:
You're right. [RFC 3261](https://www.rfc-editor.org/rfc/rfc3261) says:
```
IPv4address    =  1*3DIGIT "." 1*3DIGIT "." 1*3DIGIT "." 1*3DIGIT
IPv6reference  =  "[" IPv6address "]"
```

Since it worked on all PBX I've tested, I didn't care. But sure, I'll take a look later today. Or feel free to edit the code, I've just rebased it.

Reply:
If you don't mind, I'll leave changing the code up to you; I don't have IPv6 set up in my network right now, so I can't really test the code properly. I'll re-test with IPv4 once your changes are in.

Thanks again!

Reply:
PTAL